pages/tutorials/graphql_sso.md.erb

```diff
@@ -0,0 +1,289 @@
+# Manual SSO Setup with GraphQL
+
+- can self-serve your SSO setup with Buildkite using GraphQL
+- follow the steps below to enable sso for your org
+- set up Buildkite in your identity provider, guides available for x x and x
+
+## How to enable SSO for your Org
+
+- you need the service URL from Buildkite before you can finish setting up things in your identity provider
+
+- overview: set up a blank SSO provider in Buildkite, do the set up in your SSO provider, come back and update your blank Buildkite SSO to contain the details of your provider
+- the set up on the Buildkite side is done via a series of graphql operations
+- this guide isn't an api reference and so won't cover every available option
+- if you haven't used our graphql explorer before, check out our [api guide](http://buildkite.localhost/docs/graphql-api#getting-started)
+
+### Get your internal Organization ID
+
+- to set up your organisation for SSO, you need your internal Buildkite id
+- use the get method with your org slug to retrieve your org id
+- this id will be required in all other sso methods
+
+For example, the following code will retrieve any SSO providers that have been created for the organisation `Acme-Inc`:
+
+```graphql
+query GetProviders {
```

Comment:
@keith suggested that we add an Organization ID field to the Organization settings page, so we can drop this query.

Reply:
Sweet, I've moved the code block down the end to the 'other operations' section so people can view the providers that they've created